src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
+@mixin styles($color, $height, $width) {
+  background-color: $color;
+  height: $height;
+  width: $width;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform-origin: 50% 100%;
+}
+
+@mixin stopwatch-styles {
+  position: relative;
+  height: 80vmin;
+  width: 80vmin;
+  border: dotted 1vmin #000;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stopwatch {
+  @include stopwatch-styles;
+
+  &__dot {
+    transform: translate(-50%, -50%);
+
+    @include styles(#f6a603, 5vmin, 5vmin);
+
+    border-radius: 50%;
+  }
+
+  &__minute {
+    @include styles(#0700ff, 20vmin, 3vmin);
+
+    transform: translate(-50%, -100%);
+    animation: hand-move 3600s steps(60) infinite;
+  }
+
+  &__second {
+    @include styles(#2c8000, 38vmin, 1.5vmin);
+
+    transform: translate(-50%, -100%);
+    animation: hand-move 60s linear infinite;
+  }
+
+  &--speed-up {
+    @include stopwatch-styles;
```

Comment:
Redundant - these styles exists in `.stopwatch` so we don't need to add them again with `stopwatch--speed-up`, only change animation property for clockwatch hands

Reply:
If the dial should remain, it should not be removed, it will not be displayed without this style